src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+const Input = (props) => {
+  const ref = useRef(null);
+
+  useEffect(() => {
+    ref.current.focus();
+    ref.current.addEventListener("blur", function () {
+      props.onEditComplete(ref.current.value);
+    });
+    ref.current.addEventListener("keydown", function (event) {
+      if (event.key === "Enter") {
+        props.onEditComplete(ref.current.value);
+      }
+    });
+  }, []);
+
+  return <InputForm ref={ref} defaultValue={props.defaultValue}></InputForm>;

```

Comment:
```suggestion
  return <InputForm ref={ref} defaultValue={props.defaultValue} />;
```
こうすると閉じタグなしでかけます！
childrenを渡す必要がないコンポーネントはこうするとスッキリすると思います。